app/models/loan_response.rb

```diff
@@ -74,6 +74,10 @@ def blank?
       (embeddable_media.blank? || embeddable_media.url.blank?)
   end
 
+  def present?
+    !blank?
+  end
+
```

Comment:
`blank?` was overridden above, and I wanted to make sure `present?` worked as expected. I didn't end up using it.